Source/WebKit/NetworkProcess/storage/OriginStorageManager.cpp

```diff
@@ -658,6 +658,13 @@ void OriginStorageManager::setPersisted(bool value)
     defaultBucket().setMode(value ? StorageBucketMode::Persistent : StorageBucketMode::BestEffort);
 }
 
+WebCore::StorageEstimate OriginStorageManager::estimate()
+{
+    ASSERT(!RunLoop::isMain());
+
+    return WebCore::StorageEstimate { quotaManager().usage(), quotaManager().quota() };
```

Comment:
This is a potential privacy issue. We need to fuzzy enough the usage() so that the website cannot know the exact size of no-cors opaque responses.
The old cache storage code path is making sure of that and we need to apply similar principles.

Reply:
The usage is fuzzy as we kept the cache storage padding behavior in new implementation.
Or do you mean we also want fuzz in other types of usage?